src/Ztm.WebApi.Tests/AddressPools/EntityReceivingAddressStorageTests.cs

```diff
@@ -107,12 +115,11 @@ await this.subject.AddAsync(TestAddress.Regtest2, CancellationToken.None)
             var result = await this.subject.ListAsync(AddressFilter.None, CancellationToken.None);
 
             // Assert.
-            Assert.Contains(receivingAddresses.First(), result);
-            Assert.Contains(receivingAddresses.Last(), result);
+            Assert.Subset(result.ToHashSet(), receivingAddresses.ToHashSet());
```

Comment:
Seems like `ToHashSet` is introduced by `NBitcoin`. We should not depend on any utilities from NBitcoin that does not related to Bitcoin.

Reply:
After determine `ToHashSet` is in linq